src/internal/connector/mockconnector/mock_data_collection.go

```diff
@@ -15,14 +15,15 @@ import (
 
 // MockExchangeDataCollection represents a mock exchange mailbox
 type MockExchangeDataCollection struct {
-	fullPath     path.Path
-	messageCount int
-	Data         [][]byte
-	Names        []string
-	ModTimes     []time.Time
-	ColState     data.CollectionState
-	PrevPath     path.Path
-	DeletedItems []bool
+	fullPath        path.Path
+	messageCount    int
+	Data            [][]byte
+	Names           []string
+	ModTimes        []time.Time
+	ColState        data.CollectionState
+	PrevPath        path.Path
+	DeletedItems    []bool
+	doNotMergeItems bool
```

Comment:
nit: can you export this please? I'm going to need it when I wire up the KW side of things
```suggestion
	DoNotMergeItems bool
```

Reply:
sure can